### PR TITLE
Kshu osisoft docker installation update

### PR DESCRIPTION
**Pull request by kshuOSIsoft**
Updated dockerfile contents for creating am image

### CODE REVIEW

**Comment by TinaUll** [comment]
@kshuOSIsoft I assume this is only for v1.2?

File: content/installation/installation-using-docker.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ This topic provides examples of how to create a Docker container with the OPC UA
 	**ARM32**
 
 	```bash
-	FROM ubuntu
+	FROM ubuntu:20.04
 	WORKDIR /
-	RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends libicu60 libssl1.0.0
+	RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y ca-certificates libicu66 libssl1.1 curl
```
Comment (zryska) [inline content/installation/installation-using-docker.md:64]: We probably don't have to add curl to the image in case of all the examples

**Review by zryska — APPROVED** [review]

File: content/installation/installation-using-docker.md (hunk 1 above)
Comment (kshuOSIsoft) [reply content/installation/installation-using-docker.md:64]: wouldn't they unable to use curl to setup the configuration files then?

Comment (jfnoss) [reply content/installation/installation-using-docker.md:64]: curl is not usually necessary in a container.  It is only useful for troubleshooting a container if you want to diagnose problems with it.

**Review by TinaUll — APPROVED** [review]

**Pull request merged**